README.md

```diff
@@ -26,8 +26,8 @@ We recommend checking out the [Roam Depot Docs](https://roamresearch.com/#/app/d
 
 RoamJS manages two NPM packages, both of which have been vetted by the Roam team themselves:
 
-- [RoamJS Components](https://roamjs.com/extensions/developer/roamjs_components) - This package contains all of the UI components and utilities RoamJS uses to build extensions
-- [RoamJS Scripts](https://roamjs.com/extensions/developer/roamjs_scripts) - This package contains all of the build scripts and command line utilities RoamJS uses to build extensions
+- [RoamJS Components](https://npmjs.com/roamjs-components) - This package contains all of the UI components and utilities RoamJS uses to build extensions
+- [SamePage Scripts](https://www.npmjs.com/package/@samepage/scripts) - This package contains all of the build scripts and command line utilities RoamJS uses to build extensions
```

Comment:
I also just want to double check this is an equivalent replacement

Reply:
✅